includes/payments/functions.php

```diff
@@ -161,7 +161,7 @@ function edd_insert_payment( $payment_data = array() ) {
 			'currency'     => $payment_data['currency'],
 			'downloads'    => $payment_data['downloads'],
 			'user_info'    => $payment_data['user_info'],
-			'cart_details' => $payment_data['cart_details']
+			'cart_details' => $payment_data['cart_details'],
```

Comment:
I stand corrected, sorry didn't know that one was added to their list. Carry on.